src/kvirc/sparser/KviIrcServerParser_ctcp.cpp

```diff
@@ -826,6 +826,17 @@ void KviIrcServerParser::parseCtcpRequest(KviCtcpMessage * msg)
 {
 	msg->pData = extractCtcpParameter(msg->pData, msg->szTag);
 
+	bool bAction = KviQString::equalCI(msg->szTag, "ACTION");
+
+	if (!bAction)
```

Comment:
Why not just wrap this in to the `if` below?

`if (!bAction && IS_ME(...) && !IS_ME(...))`